Play store crash

```
java.lang.RuntimeException: An error occured while executing doInBackground()
    at android.os.AsyncTask$3.done(AsyncTask.java:300)
    at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
    at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
    at java.util.concurrent.FutureTask.run(FutureTask.java:242)
    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
    at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.StringIndexOutOfBoundsException: length=4; index=0
    at java.lang.String.indexAndLength(String.java:500)
    at java.lang.String.codePointBefore(String.java:1697)
    at fr.neamar.kiss.dataprovider.AppProvider.getResults(AppProvider.java:69)
    at fr.neamar.kiss.DataHandler.getResults(DataHandler.java:266)
    at fr.neamar.kiss.searcher.QuerySearcher.doInBackground(QuerySearcher.java:29)
    at fr.neamar.kiss.searcher.QuerySearcher.doInBackground(QuerySearcher.java:15)
    at android.os.AsyncTask$2.call(AsyncTask.java:288)
    at java.util.concurrent.FutureTask.run(FutureTask.java:237)
    ... 4 more
```

In v2.19.1.

--- a/app/src/main/java/fr/neamar/kiss/dataprovider/AppProvider.java
+++ b/app/src/main/java/fr/neamar/kiss/dataprovider/AppProvider.java
@@ -50,7 +50,8 @@ public class AppProvider extends Provider<AppPojo> {
                     }
 
                     // If we are at the beginning of a word, add it to matchedWordStarts
-                    if (normalizedAppPos == 0 || Character.isUpperCase(pojo.name.codePointAt(appPos))
+                    if (appPos == 0 || normalizedAppPos == 0
+                            || Character.isUpperCase(pojo.name.codePointAt(appPos))
                             || Character.isWhitespace(pojo.name.codePointBefore(appPos)))
                         matchedWordStarts += 1;
 
@@ -65,7 +66,8 @@ public class AppProvider extends Provider<AppPojo> {
                 }
 
                 // If we are at the beginning of a word, add it to totalWordsStarts
-                if (normalizedAppPos == 0 || Character.isUpperCase(pojo.name.codePointAt(appPos))
+                if (appPos == 0 || normalizedAppPos == 0
+                        || Character.isUpperCase(pojo.name.codePointAt(appPos))
                         || Character.isWhitespace(pojo.name.codePointBefore(appPos)))
                     totalWordStarts += 1;
 

